packages/react/src/Hero/Hero.tsx

```diff
@@ -24,8 +24,13 @@ export type HeroProps = BaseProps<HTMLDivElement> & {
 export const Hero = forwardRef<HTMLDivElement, HeroProps>(
   ({className, heading, description, primaryAction, secondaryAction, align = 'start', ...rest}, ref) => {
     return (
-      <div className={clsx(styles.Hero, styles[`Hero--align-${align}`], className)} ref={ref} {...rest}>
-        <Heading className={styles['Hero-heading']} as="h1">
+      <section
+        aria-labelledby="hero-section-brand-heading"
```

Comment:
if someone wants to customise this, could we instead make this a prop value, with `hero-section-brand-heading` being the default value?

Reply:
That's a good call. I added a new prop as suggested, with the default.